<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="47709" started-at="2021-04-22T01:27:43Z" finished-at="2021-04-22T01:28:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="47709" started-at="2021-04-22T01:27:43Z" finished-at="2021-04-22T01:28:30Z">
      <class name="com.crm.qa.testcases.ContactPageTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="setUp" is-config="true" duration-ms="21452" started-at="2021-04-21T21:27:43Z" finished-at="2021-04-21T21:28:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="verifyContactsLabelTest()[pri:1, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="verifyContactsLabelTest" duration-ms="463" started-at="2021-04-21T21:28:04Z" finished-at="2021-04-21T21:28:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyContactsLabelTest -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="TearDown" is-config="true" duration-ms="6734" started-at="2021-04-21T21:28:05Z" finished-at="2021-04-21T21:28:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="setUp" is-config="true" duration-ms="16219" started-at="2021-04-21T21:28:11Z" finished-at="2021-04-21T21:28:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="validateContactsPageTitle()[pri:2, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="validateContactsPageTitle" duration-ms="7" started-at="2021-04-21T21:28:28Z" finished-at="2021-04-21T21:28:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateContactsPageTitle -->
        <test-method status="PASS" signature="TearDown()[pri:0, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="TearDown" is-config="true" duration-ms="2776" started-at="2021-04-21T21:28:28Z" finished-at="2021-04-21T21:28:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
        <test-method status="SKIP" signature="createNewQueryTest(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:3, instance:com.crm.qa.testcases.ContactPageTest@6a4f787b]" name="createNewQueryTest" duration-ms="0" started-at="2021-04-21T21:28:30Z" data-provider="getCitywideTestData" finished-at="2021-04-21T21:28:30Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[java.lang.NullPointerException]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: java.lang.NullPointerException
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:49)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:115)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:509)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1308)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1036)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1126)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
Caused by: java.lang.NullPointerException
	at org.apache.poi.ss.usermodel.WorkbookFactory.create(WorkbookFactory.java:65)
	at com.crm.qa.util.TestUtil.getTestData(TestUtil.java:41)
	at com.crm.qa.testcases.ContactPageTest.getCitywideTestData(ContactPageTest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:55)
	at org.testng.internal.MethodInvocationHelper.invokeMethodNoCheckedException(MethodInvocationHelper.java:45)
	... 22 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createNewQueryTest -->
      </class> <!-- com.crm.qa.testcases.ContactPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
